js/src/forum/components/MarkdownToolbar.js

```diff
@@ -1,28 +1,9 @@
 import Component from 'flarum/Component';
 
-const modifierKey = navigator.userAgent.match(/Macintosh/) ? 'Meta' : 'Control';
-
 export default class MarkdownToolbar extends Component {
-  oncreate(vnode) {
-    super.oncreate(vnode);
-
-    this.attrs.setShortcutHandler(this.shortcut.bind(this));
-  }
-
   view(vnode) {
-    return <div id="MarkdownToolbar" data-for={this.attrs.for} style={{ display: 'inline-block' }}>
+    return <div class="MarkdownToolbar">
```

Comment:
Is this change from `id` to `class` necessary? Are we intentionally breaking styles that use the ID here?

Reply:
I do agree with this change, just in case theres two of these toolbars on screen at the same time, despite how ridiculously unlikely that would be

Reply:
Iirc, there isn't ID based stuff anywhere else in the composer hierarchy. I don't think it should be here either.